### PR TITLE
1.0/138 upgrade to 19

### LINKED ISSUE #138: Update base code before we start on Charity apps
## Description
Code tidy/ update?

## Plan
- [ ] Revisit backlog 
- [ ] Consider moving to/ merging with haven 0.19 I18n codebase

### PR DESCRIPTION
**Pull request by katjam**
Fixes #138 

## Description

- Upgrade to 0.19

@neontribe/contemplating-action

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
-    "elm-test": "0.18.12",
-    "elm-webpack-loader": "^4.4.0",
-    "file-loader": "^1.1.5",
-    "node-sass": "^4.6.1",
+    "autoprefixer": "^9.4.4",
+    "babel-core": "^6.26.3",
+    "copy-webpack-plugin": "^4.6.0",
+    "css-loader": "^2.0.0",
+    "elm": "^0.19.0",
+    "elm-test": "^0.19.0",
```
Comment (Nikomus) [inline package.json:16]: Should we have `elm-format` as a dependency here?

Comment (katjam) [reply package.json:16]: Yes. I think so.. Good spot!

Comment (katjam) [reply package.json:16]: Done.

**Review by Nikomus — APPROVED** [review]
Awesome! 😎

**Pull request merged**